### DISCUSSION TITLE
After a db restore I can't access my data anymore

### DISCUSSION THREAD

**Discussion by astingengo** [standardnotes/forum#3140] [General]
**Describe the issue**
I deployed, self hosted, Standardnotes server [with localstack, server, mysql and redis]

I backup my mysql database e.g.: standard_notes

I destroyed the VPS and recreated again with same version for localstack, server, mysql and redis
I then deleted the fresh created db name standard_notes, created a fresh one and imported my above backup

I can see my user there but if I try to login it will not allow me [see below error message]
If I want to create new users it will work

**Additional context**
The error that I'm receiving is:
```
Unfortunately, we couldn't handle your request. Please try again or contact our support if the error persists.
```
What do I need to do in order to be able to access old data restored from database?

**Comment by jackyzy823** [comment]
1. How do you backup and restore db?
2. Any logs from server?
